src/lib.rs

```diff
@@ -97,6 +111,13 @@ impl PGAppend<*mut i8> for pg::StringInfo {
     unsafe fn add_json(self, t: *mut i8) { pg::escape_json(self, t); }
 }
 
+impl PGAppend<*const i8> for pg::StringInfo {
```

Comment:
Could we rewrite the implementation for `*mut i8` (`impl PGAppend<*mut i8> for pg::StringInfo`) to use this? Perhaps like this?

```rust
unsafe fn add_str(self, t: *mut i8) {
    self.add_str(t as *const i8);
}
```